### PR TITLE
Add banned peers RPC

### PR DESCRIPTION
**Pull request by danield9tqh**
## Summary
Add RPC of banned peers for debugging network conditions

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish-cli/src/commands/peers/banned.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  }
+}
+
+function renderTable(content: GetBannedPeersResponse): string {
+  const columns: CliUx.Table.table.Columns<BannedPeerResponse> = {
+    identity: {
+      header: 'IDENTITY',
+      get: (row) => {
+        return row.identity
+      },
```
Comment (NullSoldier) [inline ironfish-cli/src/commands/peers/banned.ts:70]: nit, if you want this to look nicer you can set a minWidth on this so it doesn't look collapsed when its empty. I think the identifies are fixed length.

File: ironfish/src/rpc/routes/peers/getBannedPeers.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+router.register<typeof GetBannedPeersRequestSchema, GetBannedPeersResponse>(
+  `${ApiNamespace.peer}/getBannedPeers`,
+  GetBannedPeersRequestSchema,
+  (request, node): void => {
+    const peerNetwork = node.peerNetwork
+
+    if (!peerNetwork) {
+      request.end({ peers: [] })
+      return
+    }
```
Comment (NullSoldier) [inline ironfish/src/rpc/routes/peers/getBannedPeers.ts:54]: this is not nullable, you can delete this

```suggestion

```

**Review by NullSoldier — APPROVED** [review]
This is pretty close, just 1 minor comment.

File: ironfish-cli/src/commands/peers/banned.ts (hunk 1 above)
Comment (danield9tqh) [reply ironfish-cli/src/commands/peers/banned.ts:71]: Nice yea added that

File: ironfish/src/rpc/routes/peers/getBannedPeers.ts (hunk 2 above)
Comment (danield9tqh) [reply ironfish/src/rpc/routes/peers/getBannedPeers.ts]: 👍🏻

**Review by dguenther — APPROVED** [review]

**Pull request merged**